woocommerce/woocommerce-admin-test-helper
# Allow adding a new experiment manually

This PR makes the following changes.

1. Removed `source` column. 
2. Added a new form to add a new experiment manually.

### Screenshots


![Screen Shot 2022-05-20 at 2 12 27 PM](https://user-images.githubusercontent.com/4723145/169611191-657854b3-c67e-43b3-89bb-305b3e3396be.jpg)

### Detailed test instructions

1. Navigate to `Tools -> WCA Test Helper -> Experiments`
2. Add a new random experiment and confirm the new experiment appears on the table after a hard refresh.
3. Use one of the existing experiment names and change the variation and confirm the variation changes.
4. Confirm the newly added experiment is in your localStorage by checking it in `Inspector -> Application -> Local Storage` (Chrome)
5. Confirm the newly added experiment is in the `wp_options` table. 

_transient_abtest_variation_$name-of-experiment
_transient_timeout_abtest_variation_$name-of-experiment
